docs/installation.rst

```diff
@@ -123,9 +123,9 @@ specific JAX GPU installation instructions, as that is the main installation dif
 
 Della and Stellar Clusters (Princeton)
 ++++++++++++++++++++++++++++++++++++++
-These instructions were tested and confirmed to work on the Della and Stellar clusters at Princeton as of 11-6-2023.
+NOTE: For GPU jobs in Della, you should ssh into <NetID>@della-gpu.princeton.edu.
```

Comment:
I am able to work just fine on `della8` so I am unsure if this is necessary, I also can use `sbatch` to launch GPU jobs from `della8` without issue

Reply:
is this for if you want to use the GPU on the login node?

Reply:
Ok, changed it.